### ISSUE TITLE
HelloWorld example is giving no output

### ISSUE THREAD

**Issue by CedricSchmeits** [bulletphysics/bullet3#4551]
I'm interested in the bullet library so to start playing around with it I've made a copy of the HelloWorld.cpp on my system. 
I'm using Ubuntu  22.04 with the following bullet installation:
Package: libbullet-dev
Version: 3.06+dfsg-4build2

When I run the build application there is no output. After diving into the code a bit, the for loop where the `setSimulation()` method is called, `dynamicsWorld->getNumCollisionObjects()` return 0.

When looking at the ground object created in the run-in, I would expect that it would have a collisionShape. But when calling:
`body->getCollisionShape()` this returns `nullptr`

The same thing happens for the sphere that is created a little further on.

**Comment by CedricSchmeits** [comment]
Just also tried it on a Debian 12 system with bullet version: 3.24+dfsg-2

Here the same thing happens.

**Comment by cuppajoeman** [comment]
I'm assuming you're talking about this file? https://github.com/bulletphysics/bullet3/blob/master/examples/HelloWorld/HelloWorld.cpp

Otherwise post the contents of the file you're working with. I remember being able to use the `HelloWorld.cpp` program as a starting point and it working correctly, I'm running on manjaro linux.

**Comment by CedricSchmeits** [comment]
Yes I wanted to use that file as starting point of a new project. But it doesn't seem to be working

**Comment by cuppajoeman** [comment]
Personally when I'm working with bullet I just clone in bullet and build it with my project to test things out. 

I don't think I'll be able to help with Debian 12 specific issues, but maybe others can talk about their experience using bullet by installing it through their package managers.

If your willing to try the direct method, then see if you can get things running through the wiki I'm creating: https://cuppajoeman.github.io/bullet-physics-manual/Building.html

**Comment by cuppajoeman** [comment]
@CedricSchmeits did you have any luck?

**Comment by CedricSchmeits** [comment]
> @CedricSchmeits did you have any luck?

Sorry did not find the time yet to try it out. Will defiantly try it beginning next year.
